src/lib/list/selection-list.ts

```diff
@@ -97,34 +97,29 @@ export class MatListOption extends _MatListOptionMixinBase
   /** Whether the label should appear before or after the checkbox. Defaults to 'after' */
   @Input() checkboxPosition: 'before' | 'after' = 'after';
 
-  /** Value of the option */
-  @Input() value: any;
-
   /** Whether the option is disabled. */
   @Input()
   get disabled() { return (this.selectionList && this.selectionList.disabled) || this._disabled; }
   set disabled(value: any) { this._disabled = coerceBooleanProperty(value); }
```

Comment:
`: boolean`?

Reply:
Value is the actual selection option value. 

We don't want to coerce it to a boolean, again! :stuck_out_tongue:

Reply:
I'm not talking about `value`, but `disabled`.

Reply:
In `selected` we have: `set selected(value: boolean) {`

Reply:
Gotcha, fixed.

Reply:
I think you misunderstood what I mean... I mean that we have the following signature for selected: `set selected(value: boolean) { ... = coerceBooleanProperty(value); }`, so the `disabled` should follow the same thing: `set disabled(value: boolean) { ... = coerceBooleanProperty(value); }`. Did you get it now?

Reply:
Ah, ok. The highlighted code I see is both on this section:

```
/** Value of the option */
-  @Input() value: any;
```

So I was kind of guessing what you meant. Making those changes now.